client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -126,10 +128,13 @@ const Header: React.FunctionComponent = () => {
 
 	React.useEffect( () => {
 		if ( newUser && newUser.bearerToken && newUser.username && ! newSite ) {
-			handleCreateSite( newUser.username, newUser.bearerToken );
+			handleCreateSite(
+				newUser.username,
+				newUser.bearerToken,
+				selectedPlan ? selectedPlan.getStoreSlug() : 'free'
```

Comment:
```suggestion
				selectedPlan ? selectedPlan.getStoreSlug() : 'free_plan'
```

This is the spelling the store uses for the free plan